runtime/vm/createramclass.cpp

```diff
@@ -2283,6 +2299,13 @@ internalCreateRAMClassDone(J9VMThread *vmThread, J9ClassLoader *classLoader, J9C
 			}
 		}
 
+		if (J9ROMCLASS_ALLOWS_NON_ATOMIC_CREATION(romClass)) {
+			classFlags |= J9ClassAllowsNonAtomicCreation;
+		}
+		if (J9ROMCLASS_HAS_DEFAULT(romClass)) {
+			classFlags |= J9ClassIsValueType;
```

Comment:
Please add a comment that J9ClassIsValueType is being reused here intentionally.